### PR TITLE
Document Algolia extension

### PR DESCRIPTION
**Pull request by dpobel**
Preview: https://deploy-preview-791--heuristic-almeida-1a1f35.netlify.app/docs/remixed/extensions/algolia/

~~Note: https://github.com/front-commerce/developers.front-commerce.com/pull/791/files#diff-db834843913aab24eb3b13f07c68e0bb5760b88a31aa72f6ce1516d55caf8c9dR52-R57 is commented out because the feature has not been implemented yet.~~ Wrong assumptions

### CODE REVIEW

File: docs/03-extensions/algolia/how-to/standalone.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+```bash title=".env"
+FRONT_COMMERCE_ALGOLIA_APPLICATION_ID=APP_ID
+FRONT_COMMERCE_ALGOLIA_SEARCH_ONLY_API_KEY=SEARCH_ONLY_API_KEY
+FRONT_COMMERCE_ALGOLIA_INDEX_NAME_PREFIX=myproject_
+```
+
+<a name="algolia-standalone-config"></a>
+
+<!--
+TODO: this feature does not existing yet in V3
```
Comment (dpobel) [inline docs/03-extensions/algolia/how-to/standalone.mdx:53]: wrong assumption.
self note: uncomment and point to a WIP page about configuration provider

**Review by mGGk-fr — APPROVED** [review]

File: docs/02-guides/configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Configurations
```
Comment (real34) [inline docs/02-guides/configuration.mdx:2]: **nitpick:** guides being task oriented, I think the title should also be.

**suggestion:** rename to "Inject configurations", "Customize configurations" or "Override configurations"

File: docs/03-extensions/algolia/how-to/magento2.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Magento 2 backend using the Algolia Magento module."
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+## Magento 2 requirements
+
+On Magento's side, you need to
+[install and configure the Algolia module for Magento¬2](https://www.algolia.com/doc/integration/magento-2/getting-started/quick-start/?client=php#installation).
```
Comment (real34) [inline docs/03-extensions/algolia/how-to/magento2.mdx:13]:
```suggestion
[install and configure the Algolia module for Magento 2](https://www.algolia.com/doc/integration/magento-2/getting-started/quick-start/?client=php#installation).
```

File: docs/03-extensions/algolia/how-to/magento2.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+:::
+
+## Front-Commerce
+
+[After installing the Algolia
+package](/docs/remixed/extensions/algolia/#installation), you need to enable the
+corresponding extension. For that, you need to tweak your
+`front-commerce.config.ts` file like:
+
+```typescript title="front-commerce.config.ts for a Magento¬2 based project"
```
Comment (real34) [inline docs/03-extensions/algolia/how-to/magento2.mdx:41]:
```suggestion
```typescript title="front-commerce.config.ts for a Magento 2 based project"
```

**Review by real34 — APPROVED** [review]
Thanks for porting this documentation!

**Pull request merged**